.github/workflows/ci.yaml

```diff
@@ -59,19 +59,47 @@ jobs:
           CXX: g++-7
         run: |
           ./ci/install.sh
+
       - name: test shell
         run: |
           ./ci/test_shell.sh
+
       - name: test python
+        run: ./ci/test_python.sh
+      - uses: codecov/codecov-action@v2
+        with:
+          flags: test_python
+      - name: coverage erase
         run: |
-          ./ci/test_python.sh
+          source tools/activate_python.sh
+          coverage erase
+
       - name: install kaldi
         run: |
           ./ci/install_kaldi.sh
+
       - name: test utils
         run: ./ci/test_utils.sh
+      - uses: codecov/codecov-action@v2
+        with:
+          flags: test_utils
+      - name: coverage erase
+        run: |
+          source tools/activate_python.sh
+          coverage erase
```

Comment:
Could you tell me what is happened with this command?

Reply:
As far as I know, it just deletes `$(pwd).coverage.*` (a mysql DB file). I think the command is better than `rm` because it is wrapped by the official command. https://coverage.readthedocs.io/en/coverage-5.5/cmd.html#cmd-erase

Reply:
And if we don't erase the file, the sub coverage per flags will be broken (overwrapped).